ts/Stock/RangeSelector/RangeSelector.ts

```diff
@@ -1450,11 +1467,18 @@ class RangeSelector {
         delete buttonTheme.width;
         delete buttonTheme.states;
 
-
         this.buttonOptions.forEach((
             rangeOptions: RangeSelectorButtonOptions,
             i: number
         ): void => {
+            rangeOptions.text = format(rangeOptions.text, {
+                count: rangeOptions.count || 1
+            });
+
+            rangeOptions.title = format(rangeOptions.title, {
+                count: rangeOptions.count || 1
+            });
```

Comment:
Q: The same as previously: Do we need this? In `init()` we set these options with already formatted values, this seems redundant to format already formatted text.